extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3432,4 +3432,40 @@ static private function parseValue( $value, $type ) {
 		return var_export( $value, true );
 	}
 
+	/**
+	 * Returns value of a given variable for all wikis
+	 *
+	 * @param $variableId String Id of variable to get
+	 * @param $limit String Limit of rows
```

Comment:
The same here

Reply:
Not sure, because we get this from query param, so I assume everything is a string (although obviously it contains a number). Should I do parsing in such case?